CMakeLists.txt

```diff
@@ -39,7 +39,12 @@ set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 if (MSVC)
 	set_property(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR} PROPERTY VS_STARTUP_PROJECT CemuBin)
 	# floating point model: precise, fiber safe optimizations
-	add_compile_options(/EHsc /fp:precise /GT)
+	add_compile_options(/EHsc /fp:precise)
+	if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
+		set(CMAKE_STATIC_LINKER_FLAGS "${CMAKE_STATIC_LINKER_FLAGS} /llvmlibthin")
```

Comment:
Could you please add a comment explaining why this is needed? Also, prefer using target options like [`STATIC_LIBRARY_OPTIONS`](https://cmake.org/cmake/help/latest/prop_tgt/STATIC_LIBRARY_OPTIONS.html) to modifying global `CMAKE_*` variables

Reply:
[`add_link_options` ](https://cmake.org/cmake/help/latest/command/add_link_options.html#command:add_link_options) seems to exist.

Reply:
Yes, but it doesn't change options for static libraries since they are not "linked" (or that's what CMake's documentation seems to suggest)

Reply:
done

Reply:
> Yes, but it doesn't change options for static libraries since they are not "linked" (or that's what CMake's documentation seems to suggest)

yeah they aren't linked, CMake uses this term for the archiving step meaning this sets the flags to `lib.exe` on windows and `ar` on unix
guess the term is a little confusing